File: app/src/main/java/dev/hertlein/sampleproject/app/Application.java
package dev.hertlein.sampleproject.app;

import dev.hertlein.sampleproject.businesslogic.PrintService;
import dev.hertlein.sampleproject.datamodel.MessageModel;

public class Application {
    public static void main(String[] args) {
        new PrintService().print(new MessageModel("Hello World"));
    }
}

File: app/src/main/java/dev/hertlein/sampleproject/app/Application.kt
package dev.hertlein.sampleproject.app

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
internal class Application

@Suppress("SpreadOperator")
fun main(args: Array<String>) {
    runApplication<Application>(*args)
}
